Sources/COWMacros/SyntaxExtensions.swift

```diff
@@ -129,6 +137,107 @@ extension FunctionDeclSyntax {
         }
       }
     }
+    return false
+  }
+  
+  var isMutating: Bool {
```

Comment:
Explicit `internal` access control is recommended. This convention is learned from the Swift Standard Library.